### ISSUE TITLE
SC2086: Incorrect warning

### ISSUE THREAD

**Issue by orbea** [koalaman/shellcheck#1189]
#### For bugs
- Rule Id: `SC2086`
- My shellcheck version: `online`
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:
```sh

#!/bin/sh

case "$1" in
  foo|bar) echo $1 ;;
  *) echo "$1" ;;
esac

```
#### Here's what shellcheck currently says:
```
Line 4:
  foo|bar) echo $1 ;;
                ^-- SC2086: Double quote to prevent globbing and word splitting.
```
#### Here's what I wanted or expected to see:

Nothing. If `"$1"` is either `foo` or `bar` then there will never be globbing or word splitting and the warning is just erroneous.

**Comment by orbea** [comment]
Thanks for the reply and yea that is understandable. Also I didn't think about passing IFS that way, but I could always add this to a script. :)
```
# Set IFS to (space)(tab)(newline)
IFS='   
'
```
